source/SIL.AppBuilder.Portal.Frontend/src/ui/components/user-table/row/multi-role-select/display.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { Dropdown } from 'semantic-ui-react';
+
+import { isEmpty, unique } from '@lib/collection';
+
+import {
+  attributesFor,
+  OrganizationResource, UserResource,
+  RoleResource, UserRoleResource
+} from '@data';
+import { ROLE } from '@data/models/role';
+
+import { i18nProps } from '@lib/i18n';
+import { RequireRole } from '@ui/components/authorization';
+import ActiveRolesDisplay from './active-roles-display';
+import RoleSelect from './role-select';
+
+export interface IOwnProps {
+  user: UserResource;
+  organizations: OrganizationResource[];
+  roles: RoleResource[];
+  userRoles: UserRoleResource[];
+  roleNames: string;
+}
+
+export type IProps =
+& IOwnProps
+& i18nProps;
+
+export default class MultiRoleSelect extends React.Component<IProps> {
+  render() {
+    const { roles, userRoles, organizations, user, roleNames, t } = this.props;
+
+    if (isEmpty(organizations)) {
+      return t('errors.orgMembershipRequired');
+    }
+
+    return (
+      <Dropdown
+        data-test-role-multi-select
+        multiple
+        text={roleNames}
+        className='w-100 multiDropdown'
+      >
+        <Dropdown.Menu data-test-role-menu>
+          {
+            organizations.map((organization, index) => {
+              const organizationName = attributesFor(organization).name;
```

Comment:
const organizationName = attributesFor(organization).name **|| ''**; //just in case?

Reply:
naw, if we need to do that, we aren't handling asynchrony properly